.gitignore

```diff
@@ -21,3 +21,4 @@ tests/*.xyz
 *.pyc
 .autode_calculations
 tests/initial_path/*
+.mypy_cache
```

Comment:
I think this file might be missing a trailing newline?

Reply:
Good spot - thanks
```suggestion
.mypy_cache

```